db/migrate/20160617213604_create_containers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateContainers < ActiveRecord::Migration
+  def change
+    create_table :containers do |t|
+      t.string :type
```

Comment:
:type needs to be changed to something else or Rails will bark about it (:type is reserved for STI)